wd_demo.au3

```diff
@@ -1235,8 +1239,11 @@ EndFunc   ;==>SetupGecko
 
 Func SetupChrome($bHeadless)
 	_WD_Option('Driver', 'chromedriver.exe')
-	_WD_Option('Port', 9515)
-	_WD_Option('DriverParams', '--verbose --log-path="' & @ScriptDir & '\chrome.log"')
+	Local $iPort = _WD_GetFreePort(5555, 5600)
+	If @error Then Return SetError(@error, @extended, 0)
+
+	_WD_Option('Port', $iPort)
+	_WD_Option('DriverParams', '--port=' & $_WD_PORT & ' --verbose --log-path="' & @ScriptDir & '\chrome.log"')
```

Comment:
Use $iPort instead of $_WD_PORT

Reply:
take a look: [$iPort uasge](https://github.com/Danp2/au3WebDriver/pull/519/commits/44384b84d308fac3a31e53fb0077b05bc9c6cb8b)